Uplift/Models/Gym.swift

```diff
@@ -279,7 +302,7 @@ struct Facility {
 
 struct EquipmentCategory {
     let categoryName: String
-    let equipment: [Equipment]
+    var equipment: [Equipment] = []
```

Comment:
why change to `var`?

Reply:
I had changed it temporarily so that I could edit the equipment for testing purposes, will revert